dietpi/dietpi-software

```diff
@@ -1438,6 +1436,7 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_CATX[$software_id]=-1
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/webserver_stack/#nginx'
 		aSOFTWARE_DEPS[$software_id]='89'
+		aSOFTWARE_CONFLICTS[$software_id]='75 76 81 82 83 85'
```

Comment:
I guess it should be `75 76 81 82 83 84` as ID `85` is Nginx itself

Reply:
Good catch 👍.